docs/contributing/contribute-a-plugin.md

```diff
@@ -78,6 +78,11 @@ And anyone could try your plugin via the following command:
 $ pipcook plugin install your-pipcook-plugin-name
 ```
 
+## Database
+
+If your modification involves `Database` structure, you need to write a migration script under `daemon/src/migrations`.
+For more detail about it, please refer to [Migration](https://sequelize.org/master/manual/migrations.html).
```

Comment:
We may need to add documentation for daemon developers or https://github.com/alibaba/pipcook/blob/master/docs/contributing/guide-to-contributor.md?